docker-compose/mariadb-config/prod/mariadb.cnf

```diff
@@ -0,0 +1,58 @@
+[mariadb]
+
+port                  = 3306
+bind-address          = 0.0.0.0
+
+# Fine tuning
+max_connections         = 300
+connect_timeout         = 5
+wait_timeout            = 600
+max_allowed_packet      = 32M
+thread_cache_size       = 128
+sort_buffer_size        = 128M
+bulk_insert_buffer_size = 128M
+tmp_table_size          = 32M
+max_heap_table_size     = 32M
+skip-name-resolve       = 1
+innodb_log_file_size    = 2G
```

Comment:
probably still too small a innodb_log_file_size. Marko already described the mechanism, but for reference https://mariadb.com/kb/en/innodb-redo-log/#checkpoints is how to determine size.

Reply:
Thanks!
Did the calculations we seem fine, **I got 1.31 GB**
```
LOG
---
Log sequence number 12816144401716
Log flushed up to   12816144395940
Pages flushed up to 12814727890572
Last checkpoint at  12814727890572
```